run-build-functions.sh

```diff
@@ -109,13 +109,17 @@ run_yarn() {
   # The previous pattern doesn't match the end of the string.
   # This removes the flag from the end of the string.
   yarn_local="${yarn_local%--cache-folder *}"
-
-  if yarn install --cache-folder $NETLIFY_BUILD_BASE/.yarn_cache ${yarn_local:+"$yarn_local"}
+  if [ "$NETLIFY_INSTALL_JS_DEPENDENCIES" != "false" ]
   then
-    echo "NPM modules installed using Yarn"
+    if yarn install --cache-folder $NETLIFY_BUILD_BASE/.yarn_cache ${yarn_local:+"$yarn_local"}
+    then
+      echo "NPM modules installed using Yarn"
+    else
+      echo "Error during Yarn install"
+      exit 1
+    fi
   else
-    echo "Error during Yarn install"
-    exit 1
+    echo "Skipping NPM modules install"
```

Comment:
```suggestion
    echo "Skipping Node modules install"
```

Reply:
Technically those are hosted on npm. I am wondering though if some users use Yarn as a client might be confused?

Reply:
Yea, I've encountered questions about the messaging from yarn users in the past, so I think your concern is valid.

Reply:
For consistency, we should update the other instances of `NPM modules` as well in this file. Should this be done in a separate PR?

Reply:
I feel like it's a small enough change. Not too many instances of it, so I'll go ahead and make the change.